discourse-setup

```diff
@@ -352,6 +372,17 @@ ask_user_for_config() {
     local letsencrypt_status="Enter 'OFF' to disable."
   fi
 
+  read_config "DISCOURSE_MAXMIND_LICENSE_KEY"
+  local maxmind_license_key=$read_config_result
+  if [ -z $maxmind_license_key ]
+  then
+      maxmind_license_key="1234567890123456"
```

Comment:
let's use a more obvious placeholder like `INVALID_KEY`

Reply:
that's a great idea, but I was using the default value that is in web_only.yml and standalone.yml. Changing it now will be complicated, as it will need to test both for 1234567890123456 and INVALID_KEY as well as changing the two existing templates.

Reply:
ah, didn't realize it was already there!